vta/hardware/chisel/src/main/scala/core/TensorGemm.scala

```diff
@@ -116,7 +119,7 @@ class MatrixVectorCore(implicit p: Parameters) extends Module {
     val acc_o = new TensorClientData(tensorType = "acc")
     val out = new TensorClientData(tensorType = "out")
   })
-  val dot = Seq.fill(size)(Module(new DotProduct(dataBits, size)))
+  val dot = Seq.fill(size)(Module(new DotProduct(outBits, size)))
```

Comment:
Where are you specifying the other parameters? it seems that you are passing `outBits` as `inpBits`? I think we can use implicit parameters rather since we are not using `inpBits` or `wgtBits` in other places in `MatrixVectorCore`

Reply:
meant to be
val dot = Seq.fill(size)(Module(new DotProduct(inpBits, wgtBits, size)))